### DISCUSSION TITLE
What is the role of the unique_id?

### DISCUSSION THREAD

**Discussion by tanweer-mahdi** [Nixtla/statsforecast#53] [Q&A] [answered]
Hi,

I am looking forward to experiment with this amazing library. However, I have some queries:

 What is the role of _unique_id_? Also, what is the role of _ds_? In Prophet, the _ds_ column is used to denote the date/timestamp column. Does it play the same role here?

Many thanks and cheers!

**Comment by AzulGarza** [comment, marked as answer]
Hi! Thanks for your interest in the library. 😄 

The `unique_id` column defines an identifier for each time series and the `ds` column works as you explain: it denotes the date/time stamp column. The following image shows a dataframe example with two time series. Please let us know if you have more questions. :)

<img width="269" alt="image" src="https://user-images.githubusercontent.com/10517170/156458402-78af5091-0079-455b-a3c5-f3ef835576ef.png">
